.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: rust
+cache: cargo
+
+sudo: false
+
+os:
+    - linux
+    - osx
+
+rust:
+    - stable
+    - beta 
+    - nightly
+
+script:
+    - cargo build --verbose
+    - cargo test --verbose
```

Comment:
This is very useful, I will add travis CI right after we merge this file in. Just a small nit: could you please add newline at end of this file?